.github/workflows/dapr-test.yml

```diff
@@ -246,7 +251,7 @@ jobs:
         if: env.TEST_PREFIX != ''
         run: |
           make build
-          if [ "${{ env.TARGET_OS }}" = "linux" ] && [ "${{ env.TARGET_ARCH }}" != "arm" ]; then
+          if [ "${{ env.TARGET_OS }}" = "linux" ] && [ "${{ env.TARGET_ARCH }}" != "arm64" ]; then
```

Comment:
This needs to keep "arm". Mariner images are available for "arm64" but are not available for "arm"